src/nxt_socket_msg.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) Igor Sysoev
+ * Copyright (C) NGINX, Inc.
+ */
+
+#include <nxt_main.h>
+#include <nxt_socket_msg.h>
+
+
+#if (NXT_HAVE_MSGHDR_MSG_CONTROL)
+
```

Comment:
Comment with set of platforms was here originally. Is it no longer relevant or any other reason to remove it?

Reply:
The comment documented the set of platforms that has msg_control and then in `#else` it was implemented for the only one that doesn't support (Solaris). But now I removed Solaris support and added a check on auto/sockets that fails if `msg_control` is not detected, now it's required for all platforms supported. Should we still document it?

Reply:
Maybe the whole `#if (NXT_HAVE_MSGHDR_MSG_CONTROL)` can be safely removed now.

Reply:
What do you think?

Reply:
Agreed. It is already checked in configure.